### PR TITLE
HSPC-19 Support pie plots

### PR DESCRIPTION
**Pull request by inuur**
Support pie plots in PlottingTest. Store data in x, y format.

### CODE REVIEW

File: hstest/testing/plotting/drawing/drawing_type.py [hunk 1]
```diff
@@ -27,10 +27,17 @@ class DrawingType:
...
     # https://seaborn.pydata.org/generated/seaborn.barplot.html
 
+    # ------
+
+    pie = "pie"
+
+    # https://matplotlib.org/stable/gallery/pie_and_polar_charts/pie_features.html
+    # https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.plot.pie.html
+
+    # ------
```
Comment (meanmail) [inline hstest/testing/plotting/drawing/drawing_type.py:37]:
```suggestion
    # ------

    # https://matplotlib.org/stable/gallery/pie_and_polar_charts/pie_features.html
    # https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.plot.pie.html

    pie = "pie"

    # ------
```
Обычно комментарии пишут над кодом к которому оставлен комментарий

File: hstest/testing/plotting/pandas_handler.py [hunk 2]
```diff
@@ -95,34 +95,43 @@ def get_scatter_drawings_with_normalized_data(data, x, y):
 
     @staticmethod
     def get_pie_drawings_with_normalized_data(data: 'pd.DataFrame', x, y):
+        print('x', x)
+        print('y', y)
+        print('data', data)
```
Comment (meanmail) [inline hstest/testing/plotting/pandas_handler.py:100]: В продакшен коде не должно быть `print`

**Comment by aaaaaa2493** [comment]
I really recommend making so-called "universal tests" as were written for histograms [here](https://github.com/hyperskill/hs-test-python/tree/master/tests/outcomes/plot/hist/universal_tests/data_simple) and [here](https://github.com/hyperskill/hs-test-python/tree/master/tests/outcomes/plot/hist/universal_tests/data_two_pics) and for bar plots [here](https://github.com/hyperskill/hs-test-python/tree/master/tests/outcomes/plot/bar/universal_tests)

The idea is that you write one test for every single possible way you can write pie charts in matplotlib/pandas/seaborn. You just draw the same data in various different ways. The single test ensures that every possible way of drawing pie chart will result in the same data going to DrawingData object.

This way of testing revealed numerous bugs that were fixed before the release of support histograms and bar charts.

**Comment by inuur** [comment]
> I really recommend making so-called "universal tests" as were written for histograms [here](https://github.com/hyperskill/hs-test-python/tree/master/tests/outcomes/plot/hist/universal_tests/data_simple) and [here](https://github.com/hyperskill/hs-test-python/tree/master/tests/outcomes/plot/hist/universal_tests/data_two_pics) and for bar plots [here](https://github.com/hyperskill/hs-test-python/tree/master/tests/outcomes/plot/bar/universal_tests)
> 
> The idea is that you write one test for every single possible way you can write pie charts in matplotlib/pandas/seaborn. You just draw the same data in various different ways. The single test ensures that every possible way of drawing pie chart will result in the same data going to DrawingData object.
> 
> This way of testing revealed numerous bugs that were fixed before the release of support histograms and bar charts.



> I really recommend making so-called "universal tests" as were written for histograms [here](https://github.com/hyperskill/hs-test-python/tree/master/tests/outcomes/plot/hist/universal_tests/data_simple) and [here](https://github.com/hyperskill/hs-test-python/tree/master/tests/outcomes/plot/hist/universal_tests/data_two_pics) and for bar plots [here](https://github.com/hyperskill/hs-test-python/tree/master/tests/outcomes/plot/bar/universal_tests)
> 
> The idea is that you write one test for every single possible way you can write pie charts in matplotlib/pandas/seaborn. You just draw the same data in various different ways. The single test ensures that every possible way of drawing pie chart will result in the same data going to DrawingData object.
> 
> This way of testing revealed numerous bugs that were fixed before the release of support histograms and bar charts.

How it handles if 2 or more plots should be drawn at once?

**Comment by aaaaaa2493** [comment]
@inuur 
> How it handles if 2 or more plots should be drawn at once?

Then there should be 2 sets of universal tests: the first set draws all possible ways of plotting just one plot
The second set of tests should draw every possible way of drawing the same data that "contains" 2 plots

https://github.com/hyperskill/hs-test-python/tree/master/tests/outcomes/plot/bar/universal_tests

See folders "data_simple" and "data_2_pics" - that's 2 different sets of universal tests

**Comment by inuur** [comment]
@aaaaaa2493 ok, got you! Thanks for advice.

**Review by meanmail — APPROVED** [review]

**Pull request merged**